src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static final int number = 4;
+    static final int numberDifferentFigure = 5;

```

Comment:
The constant `numberDifferentFigure` should be renamed to follow the naming convention for constants (all uppercase with underscores), such as `NUMBER_OF_DIFFERENT_FIGURES`.